pxtpy/converter.ts

```diff
@@ -2387,18 +2387,30 @@ namespace pxt.py {
 
             let scopeV = lookupName(n)
             let v = scopeV?.symbol
+
+            // handle import
             if (v && v.isImport) {
                 return quote(v.name) // it's import X = Y.Z.X, use X not Y.Z.X
             }
 
             markUsage(scopeV, n);
 
             if (n.ctx.indexOf("Load") >= 0) {
-                if (v && !v.qName)
+                if (!v)
+                    return quote(getName(n))
+                if (!v?.qName) {
                     error(n, 9561, lf("missing qName"));
-                return quote(v ? v.qName! : getName(n))
-            } else
+                    return quote("unknown")
```

Comment:
is this unreachable? or is there a chance of getting "unknown" in your code?

Reply:
Should be unreachable. The previous behavior (if reachable) was emitting "", "undefined", or "null".

Ideally I'd convince the type system it's unreachable too, but it didn't appear trivial to do so.